src/app/static/src/app/components/accessibilityPage.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <div class="container mt-5">
+        <div class="row">
+            <div class="col">
+                <h1>Accessibility on MINT</h1>
+                <p>This statement applies to content published on mint.dide.ic.ac.uk</p>
+                <p>We want as many people as possible to be able to use this website. For example, that means you should
+                    be able to:</p>
+                <ul>
+                    <li>Resize your window with content being reformatted appropriately</li>
+                    <li>Adjust your text size without the site becoming less usable</li>
+                    <li>Read text easily due to sufficient contrast between foreground and background elements</li>
+                </ul>
+                <p><a href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to
+                    use if you have a disability.</p>
+                <h2>How accessible the website is</h2>
+                <p>Parts of this website may not be fully accessible. For example:</p>
+                <ul>
+                    <li>It may not be entirely navigable by keyboard</li>
+                    <li>Some charts and tables may not be correctly interpreted by screen readers</li>
+                    <li>Some form elements may lack descriptive names or labels</li>
+                </ul>
+                <h2>What we do about known issues</h2>
+                <p>We work to achieve and maintain <a href="https://www.w3.org/TR/WCAG21/">WCAG 2.1 AA standards</a>,
+                    but it is not always possible for all our
+                    content to be accessible. Where content is not accessible, we will state a reason, warn users and
+                    offer alternatives.</p>
+                <h2>Technical information about this website’s accessibility</h2>
+                <p>Imperial College London is committed to making its website accessible in accordance with the Public
+                    Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+                <p>This website is partially compliant with the <a href="https://www.w3.org/TR/WCAG21/">Web Content
+                    Accessibility Guidelines version 2.1</a> AA standard, due to the known issues listed above.</p>
+                <h2>Reporting accessibility issues</h2>
+                <p>If you need information on this website in a different format like accessible PDF, large print, easy
+                    read, audio recording or braille or if you find any accessibility issues not listed on this page
+                    then please contact accessibility@imperial.ac.uk.</p>
```

Comment:
Could make this a mailto

Reply:
Thanks. I slavishly copied non-hyperlinked version from template. But now a link!